dotnet/src/Connectors/VectorData.Abstractions/Properties/AssemblyInfo.cs

```diff
@@ -0,0 +1,3 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+[assembly: System.Resources.NeutralResourcesLanguage("en-US")]
```

Comment:
Added this because of a compiler warning, since this PR adds string resources.

Reply:
subjective: we could add `<NeutralLanguage>en-US</NeutralLanguage>` to the project file instead of creating a new file

Reply:
No strong feeling either way... I'm used to having AssemblyInfo.cs forr assembly-wide attributes but am good for doing it via the csproj (is this what you see in e.g. runtime projects?)